model/dashboards.go

```diff
@@ -20,41 +25,57 @@ type MonitoringDashboard struct {
 
 // Chart is the model representing a custom chart, transformed from charts in MonitoringDashboard k8s resource
 type Chart struct {
-	Name      string                     `json:"name"`
-	Unit      string                     `json:"unit"`
-	Spans     int                        `json:"spans"`
-	ChartType *string                    `json:"chartType,omitempty"`
-	Min       *int                       `json:"min,omitempty"`
-	Max       *int                       `json:"max,omitempty"`
-	Metric    []*SampleStream            `json:"metric"`
-	Histogram map[string][]*SampleStream `json:"histogram"`
-	Error     string                     `json:"error"`
-}
-
-func FillHistogram(from prometheus.Histogram, scale float64, name string, chart *Chart) {
-	stats := make(map[string][]*SampleStream, len(from))
-	for stat, metric := range from {
-		if metric.Err != nil {
-			chart.Error = fmt.Sprintf("error in metric %s/%s: %v", name, stat, metric.Err)
+	Name      string          `json:"name"`
+	Unit      string          `json:"unit"`
+	Spans     int             `json:"spans"`
+	ChartType *string         `json:"chartType,omitempty"`
+	Min       *int            `json:"min,omitempty"`
+	Max       *int            `json:"max,omitempty"`
+	Metrics   []*SampleStream `json:"metrics"`
```

Comment:
Here it seems updated correctly, so all metrics are under the Metrics array as expected.

Reply:
No, I'm wrong, these are the metrics raw data.